openfe/tests/protocols/test_openmm_rfe_slow.py

```diff
@@ -84,7 +90,9 @@ def test_openmm_run_engine(benzene_vacuum_system, platform,
         nc = pur.outputs['nc']
         assert nc == unit_shared / "simulation.nc"
         assert nc.exists()
-        assert (unit_shared / "structural_analysis.json").exists()
+        structural_analysis_file = unit_shared / "structural_analysis.npz"
```

Comment:
Coverage is going to fail because we don't run these tests until merge.

Can confirm that this passes locally.

Reply:
Would it make sense to also test the failure of the `structural_analysis`?

Reply:
Done, thanks!